# roslaunch pf_drive ackermann_wbt_teach.launch path:="/home/gralerfics/MyFiles/Workspace/tr_data/car_1"
tr:
    camera:
        raw_image_topic: /car/camera/image
        processed_image_topic: /tr/camera/processed_image # debug
    odometry:
        odom_topic: /car/odom
        processed_odom_topic: /tr/odometry/processed_odom # debug
    controller:
        cmd_vel_topic: /car/cmd_vel
        k_rho: 0.35 # 0.5 # k_rho > 0
        k_alpha: 0.3 # 5.0 # 0.05 # k_alpha > k_rho
        k_beta: -1.0 # -3.0 # -0.6 # k beta < 0
        k_theta: 4.0 # 7.0 # 5.0
        velocity_min: 0.0
        velocity_max: 3.5
        omega_min: 0.0
        omega_max: 0.35 # 0.78
        translation_tolerance: 0.05
        rotation_tolerance: 0.1
    global_locator:
        type: 'webots'
        fixed_frame_id: 'map'
        supervisor_srv: '/car/supervisor'
        robot_def: 'car'
    repeater:
        match_offset_radius: 90 # px
        along_path_radius: 2
        k_rotation: 0.06 # 0.04
        k_along_path: 0.1 # 0.03
        distance_threshold: 0.2
        angle_threshold: 0.1 # 0.05
        goal_advance_distance: 10.0